net/bind/patches/010-fix-variable-name-in-conditional-block.patch

```diff
@@ -0,0 +1,25 @@
+From 261c84d91d1b4581df9f7f0ec031908299de7726 Mon Sep 17 00:00:00 2001
+From: Mark Andrews <marka@isc.org>
+Date: Thu, 19 Dec 2019 09:27:44 +1100
+Subject: [PATCH] fix variable name in conditional block
```

Comment:
Could you document the origin of this patch? A [link](https://gitlab.isc.org/isc-projects/bind9/commit/261c84d91d1b4581df9f7f0ec031908299de7726) to upstream version control, maybe? I don't think OpenWRT has a specific policy of following something like Debian's [patch tagging guidelines](https://dep-team.pages.debian.net/deps/dep3/), but I think borrowing the "Origin" header from that would be appropriate.

Reply:
Sure, why not.